src/molecules/ProfileDropdown.tsx

```diff
@@ -5,41 +5,53 @@ import GenericDropdown, {
 } from '../components/GenericDropdown';
 import styled from 'styled-components';
 import { useSignedIn } from 'hooks/useSignedIn';
+import { Link } from 'react-router-dom';
 
 const Button = styled.div`
+	padding: 1rem;
+`;
+
+const SignInLink = styled(Link)`
+	display: block;
+	text-decoration: none;
 	padding: 1em;
+	color: ${({ theme }) => theme.colors.text[2]};
+	&:hover {
+		color: ${({ theme }) => theme.colors.primary[0]};
+	}
 `;
 
 function ProfileDropdown() {
 	const { signedIn, signOut } = useSignedIn();
 	return (
-		<GenericDropdown
-			buttonDisplay={
-				<Button>
-					{'"Name of User"  '}
-					<i className="fas fa-user-circle" />
-				</Button>
-			}
-		>
-			<ListLink to="/profile">
-				<i className="fas fa-user-circle" />
-				{'Profile'}
-			</ListLink>
-			<ListLink to="/settings">
-				<i className="fas fa-cog" />
-				{'Settings'}
-			</ListLink>
-			{(signedIn && (
-				<ListButton onClick={signOut}>
-					<i className="fas fa-sign-out-alt" /> {'Sign Out'}
-				</ListButton>
-			)) || (
-				<ListLink to="/sign-in">
+		<>
+			{signedIn ? (
+				<GenericDropdown
+					buttonDisplay={
```

Comment:
I think because the sign in button component is significantly smaller than the Dropdown, we should put it first, maybe `!signedIn ? <SignIn ... /SignIn> : < Dropdown ... /Dropdown>`. I think it will help make the two states more obvious.